private/buf/cmd/buf/command/breaking/breaking.go

```diff
@@ -219,6 +219,12 @@ func run(
 		if flags.ExcludeImports {
 			breakingOptions = append(breakingOptions, bufcheck.BreakingWithExcludeImports())
 		}
+		if len(imageWithConfig.PluginConfigs()) > 0 {
+			// TODO: remove this as part of publicly releasing lint/breaking plugins
+			if err := bufcli.CheckBetaPluginEnabled(container); err != nil {
```

Comment:
You've only gated two places that call a `bufcheck.Client`, there's 6-7 in this codebase.